### ISSUE TITLE
Comment sign tortious interference

### ISSUE THREAD

**Issue by flexoron** [infradig/trealla#385]
```
Trealla Prolog (c) Infradig 2020-2021, v1.19.20
?- [user].
c :- d.
d :- write(%d).
   true.
?- listing.
c :- d.
   true.
?-
```

**Comment by flexoron** [comment]
```
v1.19.23

?- [user].
d :- write(%d).
Error: syntax error, incomplete statement
   true.

% happens if faulty rule is last line of multi-line
?- [user].
c.
d :- write(%d).
   true.
?-
```

**Comment by infradig** [comment]
That's weird.

**Comment by flexoron** [comment]
fixed with v1.19.27
```
?- [user].
c.
d :- write(%d).
Error: syntax error, incomplete statement, line 3 'd :- write(d%).
'
   true.
?- c.
uncaught exception: error(existence_error(procedure,c/0),c/0)
?- halt.
```
